CHANGELOG.md

```diff
@@ -8,24 +8,36 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [1.4.0/0.29.0] - 2022-02-11
```

Comment:
needs to be updated of course 😉

Reply:
@MrAlias You are fast!

Reply:
I've been working on my precognition skills :laughing: